IdentityCore/src/telemetry/MSIDTelemetryBaseEvent.m

```diff
@@ -134,6 +134,21 @@ - (void)addDefaultProperties
     [_propertyMap addEntriesFromDictionary:[[self class] defaultParameters]];
 }
 
++ (NSArray<NSString *> *)propertiesToAggregate
+{
+    static dispatch_once_t once;
+    static NSArray *names = nil;
+    
+    dispatch_once(&once, ^{
+        names = @[
+                  MSID_TELEMETRY_KEY_REQUEST_ID,
```

Comment:
don't we also want to aggregate metadata events? e.g. x-client-ver etc

Reply:
Metadata events are always there, see [here](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/20093b9fc7a9313ecf66dd3f601e15af7135f549/IdentityCore/src/telemetry/MSIDAggregatedDispatcher.m#L64).